packages/shared-ux/chrome/navigation/src/ui/navigation.stories.tsx

```diff
@@ -6,39 +6,41 @@
  * Side Public License, v 1.
```

Comment:
This file should be in the `src/ui/components` folder

Reply:
I will leave it here for now as it both show examples of the `Navigation` component (inside "ui/components") and `DefaultNavigation` component (at the root of "ui" folder)